hw/top_earlgrey/lint/ascentlint/top_earlgrey_lint_cfgs.hjson

```diff
@@ -14,127 +14,127 @@
   use_cfgs: [{    name: aes
                   fusesoc_core: lowrisc:ip:aes
                   import_cfgs: ["{proj_root}/hw/lint/data/common_lint_cfg.hjson"]
-                  rel_path: "hw/ip/aes/lint"
+                  rel_path: "hw/ip/aes/lint/ascentlint"
```

Comment:
Move this file to `hw/top_earlgrey/lint` and change this to `rel_path: "hw/ip/aes/lint/{tool}`. That way you only need one copy. Set `rel_path` for the summary accordingly as well.

Reply:
ok, makes sense, thanks.